Code/src/main/java/nl/utwente/group10/ui/Main.java

```diff
@@ -31,8 +31,8 @@ public void start(Stage stage) throws Exception {
         // Init TactilePane
         CustomUIPane tactilePane = new CustomUIPane(catalog);
         tactilePane.setBordersCollide(true);
-        tactilePane.setMinWidth(50000);
-        tactilePane.setMinHeight(50000);
+        tactilePane.setMinWidth(3000);
+        tactilePane.setMinHeight(3000);
```

Comment:
I'm not sure this is right: the minWidth/minHeight should be equal to the maxWidth/maxHeight if we want to prevent JavaFX from resizing the pane to its contents. You should probably either change both, or neither.